web/src/components/vis/index.js

```diff
@@ -0,0 +1,28 @@
+import WilcoxonPlotTile from './WilcoxonPlotTile.vue';
+import AnovaTableTile from './AnovaTableTile.vue';
+import { wilcoxon_zero_methods, wilcoxon_alternatives, plot_types } from '../../utils/constants';
+
+export const analyses = [
+  {
+    path: 'wilcoxon', // same as key
```

Comment:
I don't understand what the comment means.

Reply:
removed.